git-multimail/README

```diff
@@ -362,9 +362,30 @@ multimailhook.mailer
         Timeout in seconds.
 
       multimailhook.smtpEncryption
-        Set the security type. Allowed values: none, ssl, tls.
+        Set the security type. Allowed values: none, ssl, tls (starttls).
         Default=none.
 
+      multimailhook.smtpCACerts
+        Set the path to a list of trusted CA certificate to verify the
+        server certificate, only supported for tls smtpEncryption. If
+        unset or empty, the server certificate is not verified. If it
+        targets a file containing a list of trusted CA certificates (PEM
+        format) these CAs will be used to verify the server certificate.
+        For debian, you can set ``/etc/ssl/certs/ca-certificates.crt``
+        for using the system trusted CAs. For self-signed server, you
+        can add your server certificate to the system store:
+
+            cd /usr/local/share/ca-certificates/
+            openssl s_client -starttls smtp \
```

Comment:
This is weirdly formatted (see https://github.com/nononame/git-multimail/tree/smtplib-cert-required-sign-off ). The proper way to include piece of code in RST is to use a `::` right before it. I can do that when applying if you don't want to fix it (I guess I've been bothering you enough on this PR ;-) ).